### PR TITLE
Fix warnings on publish

### PR DESCRIPTION
**Pull request by ADawesomeguy**
tbh no clue what I'm doing but I looked at the compiler warnings and did what they said to do to fix them because they were an eyesore.

also for some reason rider did some weird stuff with whitespace but it seems to look ok.

### CODE REVIEW

File: WolvenKit.RED4.Types/ClassesExt/Properties/cpGameplayLightComponent.cs [hunk 1]
```diff
@@ -4,10 +4,10 @@ public partial class cpGameplayLightComponent
     {
         [OrdinalOverride(Before = 9)]
         [RED("placedEditorData")]
-        public CHandle<entIPlacedComponentEditorData> PlacedEditorData
+        public new CHandle<entIPlacedComponentEditorData> PlacedEditorData
         {
             get => GetPropertyValue<CHandle<entIPlacedComponentEditorData>>();
-            set => SetPropertyValue<CHandle<entIPlacedComponentEditorData>>(value);
+            set => SetPropertyValue(value);
```
Comment (seberoth) [inline WolvenKit.RED4.Types/ClassesExt/Properties/cpGameplayLightComponent.cs:10]: Even if it isn't needed, please revert so it matches the rest again :)

**Review by seberoth — CHANGES_REQUESTED** [review]

File: WolvenKit.RED4.Types/ClassesExt/Properties/cpGameplayLightComponent.cs (hunk 1 above)
Comment (ADawesomeguy) [reply WolvenKit.RED4.Types/ClassesExt/Properties/cpGameplayLightComponent.cs:10]: Hm I don't remember doing that. Might've been some automatic fix by Rider or something I clicked. I'll make sure to revert it.

**Review by seberoth — APPROVED** [review]
Looks good to me

**Comment by ADawesomeguy** [comment]
@rfuzzo mergeable?

**Pull request merged**